Example/GiniVision/ScreenAPIViewController.swift

```diff
@@ -99,6 +99,7 @@ class ScreenAPIViewController: UIViewController {
         
         cancelAnalsyis()
         imageData = data
+        print("Analysing document with size \(Double(data.count) / 1024.0)")
```

Comment:
Unfortunately, this is how we log in the whole library. Which is an oversight - we need to come up with a proper logging solution. In the meantime, it would probably be annoying for customers to always see a log message about file sizes. I'll surround that with an if.